doc/HttpLuaModule.wiki

```diff
@@ -5140,17 +5140,24 @@ This feature was first introduced in the <code>v0.3.1rc22</code> release.
 See also [[#ngx.shared.DICT|ngx.shared.DICT]].
 
 == ngx.shared.DICT.incr ==
-'''syntax:''' ''newval, err = ngx.shared.DICT:incr(key, value)''
+'''syntax:''' ''newval, err, forcible? = ngx.shared.DICT:incr(key, value, init?)''
 
 '''context:''' ''init_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.*, balancer_by_lua*, ssl_certificate_by_lua*''
 
 Increments the (numerical) value for <code>key</code> in the shm-based dictionary [[#ngx.shared.DICT|ngx.shared.DICT]] by the step value <code>value</code>. Returns the new resulting number if the operation is successfully completed or <code>nil</code> and an error message otherwise.
 
-The key must already exist in the dictionary, otherwise it will return <code>nil</code> and <code>"not found"</code>.
+When the key does not exist in the dictionary and if the <code>init</code> argument
+
+# is not specified, it will return <code>nil</code> and <code>"not found"</code>,
+# is specified, it will create a new <code>key</code> with the new value: <code>value + init</code>.
+
+Like the [[#ngx.shared.DICT.add|add]] method, it also overrides the (least recently used) unexpired items in the store when running out of storage in the shared memory zone.
+
+The <code>forcible</code> return value will not be returned when <code>init</code> argument is not specified.
 
```

Comment:
I think the `forcible` return value needs some documentation about its semantics.